.github/workflows/ci.yml

```diff
@@ -6,14 +6,20 @@ on:
   pull_request: ~
 
 jobs:
-  # derived from https://dev.to/felipetofoli/github-actions-for-net-full-framework-build-and-test-299h
+  Style:
+    runs-on: windows-2019
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check dotnet Style
+        run: dotnet-format --check --exclude /
```

Comment:
What's with the `--exclude /` bit?

Reply:
Pulled from how the GitHub `super-linter` action runs `dotnet-format`: https://github.com/github/super-linter/blob/d07fe8795199be429572017d54072b3312ce3c38/lib/linter.sh#L869

Excludes anything in the root directory (any files that aren't part of the `EasyPost.sln` project), such as `LayoutExample.cs`

Reply:
Perfect!